lms/djangoapps/course_structure_api/v0/serializers.py

```diff
@@ -1,4 +1,7 @@
-""" Django REST Framework Serializers """
+""" Django REST Framework Serializers
+
+TODO: delete me once grading policy is implemented in course_api.
```

Comment:
FYI - the grading policy will most likely be moved to the grades/api instead.  
I am creating a separate PR that copies the grading-policy endpoint to the new grades/api.  Once insights is updated to use that new endpoint, we can delete all this code (soon hopefully).